chat/service.py

```diff
@@ -29,15 +29,20 @@ def get_system_instruction(history_id: str) -> str:
         if mission is not None:
             ai_prompt = getattr(mission, "ai_prompt", "") or ""
             question_text = getattr(mission, "question_text", "") or ""
+            answer_assets = getattr(mission, "answer_assets", "") or ""
+            question_hint = getattr(mission, "question_hint", "") or ""
```

Comment:
❗ 순서를 question_hint를 먼저, answer_assets를 그 다음으로 해주세요.
```suggestion
            question_hint = getattr(mission, "question_hint", "") or ""
            answer_assets = getattr(mission, "answer_assets", "") or ""
```